tests/test_classifier/test_classification_report.py

```diff
@@ -128,9 +133,9 @@ def test_pandas_integration(self):
         viz.fit(X_train, y_train)
         viz.score(X_test, y_test)
 
-        self.assert_images_similar(viz, tol=0.1)
+        self.assert_images_similar(viz, tol=43.0)
```

Comment:
This is a fairly big jump in tolerance; could we also create an issue to review why these high tolerances are required? Maybe just note the classification report in #379 has pretty high tolerances and they should be reviewed.

Reply:
@ndanielsen 
These tolerances were set at sprints and I didn't closely monitor them thus they might be artificially inflated.

Reply:
Awesome, the issue you created is perfect.